### PR TITLE
Add the MySQL compatibility document

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

**Comment by queenypingcap** [comment]
@shenli @c4pt0r @siddontang PTAL.

File: op-guide/mysql-compatibility.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Compatibility with MySQL
+
+TiDB supports the majority of the MySQL grammar, including cross-row transactions, JOIN, subquery, and so on. You can connect to TiDB directly using your own MySQL client. If your existing business is developed based on MySQL, you can replace MySQL with TiDB to power your application without changing a single line of code in most cases.
+
+TiDB is compatible with most of the MySQL database management & administration tools such as `PHPMyAdmin`, `Navicat`, `MySQL Workbench`, and so on. It also  support the database backup tools, such as `mysqldump` and `mydumper/myloader`.
```
Comment (siddontang) [inline op-guide/mysql-compatibility.md]: supports

File: op-guide/mysql-compatibility.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+However, in TiDB, there are some MySQL features that are not supported or are different because they cannot be implemented well in distributed environments.
```
Comment (shenli) [inline op-guide/mysql-compatibility.md]: environments -> scenario

**Comment by c4pt0r** [comment]
LGTM

**Pull request merged**